### ISSUE TITLE
1.15.2: Can't get color gui permissions right

### ISSUE THREAD

**Issue by jnijland** [kangarko/ChatControl-Pro#1803]
I'm trying to get the permissions for the /chatcontrol color menu right (on a 1.15.2 server running luckperms and chatcontrol 8.7.1), to enable my users to receive a chatcolor via a crate, but I can't seem to get the permissions set up right. 

I gave the following permissions to the default user group: 
```chatcontrol.command.color,chatcontrol.command.color.gui.7,chatcontrol.chat.format.color.7```

I gave my specific user the following permissions:
```chatcontrol.command.color.gui.f,chatcontrol.chat.format.color.f```

The GUI opens for the user, but when the user selects a color (in this case white), nothing happens. 

Any idea what's going wrong here? I don't expect it to be a bug, just some misconfiguration from my side.

**Comment by AngellitoX** [comment]
Hello there!

The color GUI requires a series of permissions which all need to be setup in order to work.

Please follow these steps to know if you are using LuckPerms:
1. Type /lp record chatcontrol
2. Have an user to open the color gui and try to use a color.
3. Once he does that (and fails), type /lo verbose paste
4. Open the link and look for the permissions that are missing to be added.

Hope this helps for you! Consider reopening if your issue persists.
Angel from MineAcademy.org

**Comment by jnijland** [comment]
Hi there, 

I've tried for a while, but I still can't seem to get the permissions for /chatcontrol color working for my players. I've made a verbose record for both me (as OP) and a default player, compared the records and came to the conclusion that they now have the same permissions:
```chatcontrol.command.color,chatcontrol.command.color.gui,chatcontrol.command.color.gui.gray,chatcontrol.command.color.gui.white```

I guess there is a bug somewhere in the permissions for this command. I really tried every possible combination now.
